README.md

```diff
@@ -523,6 +526,31 @@ go tool pprof http://localhost:6060/debug/pprof/heap
 
 A web page interface is provided by http/pprof at http://localhost:6060/debug/pprof/.
 
+### Fuzzing
+
+Fuzz tests are run with [go-fuzz](https://github.com/dvyukov/go-fuzz).
+[Follow the instructions on the go-fuzz page](https://github.com/dvyukov/go-fuzz) to install it.
+
+Fuzz tests are written for the following packages:
+
+#### base58
+
+To fuzz the `cipher/base58` package,
+
+```sh
+go-fuzz-build github.com/skycoin/skycoin/src/cipher/base58/internal
```

Comment:
It'd be nice to have a make target to fuzz all possible packages , as well as those in a comm-separated list (`Makefile` parameter?)

Reply:
Added